common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java

```diff
@@ -52,6 +52,7 @@
  * </p>
  *
  * @author Bryan Baugher
+ * @deprecated Use Kafka's new AdminClient
```

Comment:
You are failing the build...

> [INFO] There is 1 error reported by Checkstyle 6.11.2 with checkstyle/checkstyle-config.xml ruleset.
[ERROR] src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java:[57] (annotation) MissingDeprecated: Must include both @java.lang.Deprecated annotation and @deprecated Javadoc tag with description.